client/webserver/site/src/html/markets.tmpl

```diff
@@ -391,6 +394,13 @@
           <button id="vSubmit" type="button" class="w-75 mt-1 justify-content-center fs15 bg2 selected">Submit</button>
         </div>
       </div>
+      <hr class="dashed my-4">
+      <div class="disclaimer mt-3 fs14">
+        <span class="red">IMPORTANT</span>: Trades take time to settle, and you cannot turn off the DEX client software, or
+        the <span data-unit="quote"></span> or <span data-unit="base"></span> blockchain daemons, until settlement is complete.
+        Settlement can complete as quickly as a few minutes or as long as a few hours.
```

Comment:
The average user probably won't know what "daemons" is talking about.  I'd be fine calling it "wallet software (e.g. dcrwallet and bitcoind)" so there is no confusion.

In a second one-sentence paragraph, this might be a good place to remind users to ensure their blockchains are done syncing, although we'll have to check that discussed.

Reply:
You mind if I leave this in anticipation of #785?